src/middleware/basetypes.rs

```diff
@@ -33,7 +33,10 @@ use std::{
 
 use hex::{FromHex, FromHexError, ToHex};
 use plonky2::{
-    field::types::{Field, PrimeField64},
+    field::{
+        goldilocks_field::GoldilocksField,
```

Comment:
We rename this type as `F` here https://github.com/0xPARC/pod2/blob/143a8c9d4eb43046ef1c24d4df0a7fc8b0fc647c/src/backends/plonky2/basetypes.rs#L14

And then re-export it in the middleware here https://github.com/0xPARC/pod2/blob/143a8c9d4eb43046ef1c24d4df0a7fc8b0fc647c/src/middleware/basetypes.rs#L49

To be consistent could you remove this import and rewrite the `SELF_ID_HASH` as `[F(0x5), F(0xe), ...`?